pkgs/by-name/be/betterdisplaycli/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  stdenv,
+  swift,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "betterdisplaycli";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "waydabber";
+    repo = "betterdisplaycli";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rfJDoPzIsBaKkGhgsGciErAoDycMDojxjYnT+oQZjnA=";
+  };
+
+  buildInputs = [ swift ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp betterdisplaycli $out/bin
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Simple CLI access for BetterDisplay";
+    homepage = "https://github.com/waydabber/betterdisplaycli";
+    license = lib.licenses.mit;
+    mainProgram = "betterdisplaycli";
+    maintainers = [ ];
+    platforms = [ "aarch64-darwin" ];
```

Comment:
Does this work on intel? If so `platforms = lib.platforms.darwin` would be a better option

Ofborg will run the build if you don't have an intel machine to test on